cpp/ql/test/library-tests/rangeanalysis/rangeanalysis/RangeAnalysis.expected

```diff
@@ -0,0 +1,2264 @@
+instructionBounds
```

Comment:
These expected results are too long to review, and probably too long even to review a diff properly alongside future changes. Can we make a test that reports the ranges only of particular expressions of interest to a human - e.g. assignments to a variable with a particular name?

Reply:
The new version of the test looks much better, thanks.